FEXCore/Source/Utils/Telemetry.cpp

```diff
@@ -61,11 +61,11 @@ namespace FEXCore::Telemetry {
     auto DataDirectory = Config::GetDataDirectory();
     DataDirectory += "Telemetry/" + ApplicationName + ".telem";
 
-    if (FHU::Filesystem::Exists(DataDirectory)) {
-      // If the file exists, retain a single backup
-      auto Backup = DataDirectory + ".1";
-      FHU::Filesystem::CopyFile(DataDirectory, Backup, FHU::Filesystem::CopyOptions::OVERWRITE_EXISTING);
-    }
+    // Retain a single backup if the telemetry already existed.
+    auto Backup = DataDirectory + ".1";
```

Comment:
Minor nit, but:

```suggestion
    auto Backup = DataDirectory + ".bck";
```

(or .backup, if we want to be super literal)

Maybe, just to make it a little more self-evident it's a backup.

Reply:
Sure, renamed to bck. Numbered suffix is just a linux-ism that I'm not beholden to.